src/span.js

```diff
@@ -155,7 +155,8 @@ export default class Span {
      **/
     finish(finishTime: ?number): void {
         if (this._duration !== undefined) {
-            throw new Error('You can only call finish() on a span once.');
+            this.tracer()._logger.error('You can only call finish() on a span once.');
```

Comment:
nit: the error should provide better context. At minimum log the operation name so that people know which span is being double-finished.